tests/Authentication/Filters/SessionFilterTest.php

```diff
@@ -83,4 +83,12 @@ public function testBlocksInactiveUsers(): void
 
         setting('Auth.actions', ['register' => null]);
     }
+
+    public function testStoreRedirectsToEntraceUrlIntoSession(): void
+    {
+        $result = $this->call('get', 'protected-route');
+        $result->assertRedirectTo('/login');
+        $this->assertNotEmpty($_SESSION['beforeLogginUrl']);
+        $this->assertSame(site_url('protected-route'), $_SESSION['beforeLogginUrl']);
```

Comment:
All superglobals should be removed. So please use the Session class.
```suggestion

        $result->assertRedirectTo('/login');

        $session = session();
        $this->assertNotEmpty($session->get('beforeLogginUrl'));
        $this->assertSame(site_url('protected-route'), $session->get('beforeLogginUrl'));
```